### PR TITLE
Formula-Cookbook: add `std_npm_args`

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
Adds the new `std_npm_args` to the Formula Cookbook.

### CODE REVIEW

**Comment by p-linnane** [comment]
Style was failing unless I put `require "language/node"` in single quotes.

File: docs/Formula-Cookbook.md [hunk 1]
```diff
@@ -514,6 +514,16 @@ The `std_*_args` methods, as well as the arguments they pass, are:
...
 ```
 
+#### `std_npm_args`
+
+```ruby
+"require 'language/node'"
+
+"return Language::Node.std_npm_install_args(Pathname(prefix)) if prefix"
+
+"Language::Node.local_npm_install_args"
```
Comment (branchvincent) [inline docs/Formula-Cookbook.md:524]: I would list what these actually resolve to:

```ruby
"-ddd"
"--global"
"--build-from-source"
"--cache=$(brew --cache)/npm_cache"
"--prefix=#{libexec}"
"$PWD/$(npm pack 2>/dev/null)"
```

not sure the best way to express that last line though

**Review by branchvincent — APPROVED** [review]

**Review by MikeMcQuaid — COMMENTED** [review]
I appreciate this critique should have probably landed in the prior PR but: my concern with documentation like this is it inevitably results in the docs and the code becoming out of sync. What would you think about adjusting the docs to link to the code instead?

**Comment by p-linnane** [comment]
@MikeMcQuaid So remove the actual listed args, and just have a list of `std_*_args` that are links to the lines in the codebase?

**Comment by carlocab** [comment]
> @MikeMcQuaid So remove the actual listed args, and just have a list of `std_*_args` that are links to the lines in the codebase?

Or the online documentation, e.g. https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method. Some of it sucks though (e.g. https://rubydoc.brew.sh/Formula.html#std_cmake_args-instance_method)

**Review by carlocab — APPROVED** [review]
I'm ok for feedback to be fixed in succeeding PRs.

**Pull request merged**